parachain/pallets/basic-channel/src/outbound/mod.rs

```diff
@@ -30,14 +30,22 @@ pub use weights::WeightInfo;
 )]
 #[scale_info(skip_type_params(M, N))]
 #[codec(mel_bound())]
-pub struct MessageBundle<M: Get<u32>, N: Get<u32>> {
+pub struct MessageBundle<M: Get<u32>, N: Get<u32>, AccountId> {
 	source_channel_id: u8,
+	account: AccountId,
 	/// Unique nonce for to prevent replaying bundles
 	#[codec(compact)]
 	nonce: u64,
-	messages: BoundedVec<Message<M>, N>,
+	messages: BoundedVec<EnqueuedMessage<AccountId, M>, N>,
 }
 
+#[derive(
+	Encode, Decode, CloneNoBound, PartialEqNoBound, RuntimeDebugNoBound, MaxEncodedLen, TypeInfo,
+)]
+#[scale_info(skip_type_params(AccountId, M))]
```

Comment:
`<T as frame_system>::AccountId` likely already implements all those traits (Encode, Decode, etc), so you may not have to specify it in `skip_type_params`.

Reply:
More info on those `mel_bound`, `skip_type_params` macros:
https://substrate.stackexchange.com/questions/1423/what-is-the-role-of-scale-infoskip-type-paramst
https://substrate.stackexchange.com/questions/1974/how-does-codecmel-bound-work/1977#1977